static/styles/night_mode.css

```diff
@@ -233,6 +233,25 @@ on a dark background, and don't change the dark labels dark either. */
         border-color: hsla(0, 0%, 0%, 0.9);
     }
 
+    /* custom scrollbar for new and preview message_textarea */
```

Comment:
I'm not sure placing this here is the right solution. For instance, we also have a `light theme` which won't be covered by your changes.

Reply:
I was a bit reluctant to change any style outside night-theme as I thought it was out of scope of this issue (I expected to change style only in night theme)
I will make changes very soon and update the PR

Reply:
Also should I change styles for the page scrollbar (right-most end) too or it would be better to do it in a different issue?
![Screenshot from 2020-10-12 23-30-12](https://user-images.githubusercontent.com/57325543/95777197-53ebbd80-0ce3-11eb-9f3a-9fb148db5884.png)

Reply:
You could place this in `renderd_markdown.css` which has most of our scroll-bar stylings. `zulip.css` is another option I suppose, but maybe more of a last resort place.

Re main view styling: I think that's outside the scope of this issue. You could bring it up on czo and post some screenshots there for feedback.